spec/controller/patients_controller_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe PatientsController, type: :controller do
+  describe "GET #index" do
+    before{get :index}
+    let!(:accounts){FactoryBot.create_list :account, 5}
```

Comment:
khác nhau giữa ! và k có ! là gì e nhỉ

Reply:
let là khởi tạo đối tượng 1 lần trong 1 it, không thay đổi được. let chỉ chạy khi và chỉ khi được gọi đến
let! khởi tạo đối tượng trong 1 khối before ngầm, trước khi it được thực thi. let! hữu ích khi cần định nghĩa những trạng thái trước khi khối it được thực thi.